### PR TITLE
fitlering B-C

### PR DESCRIPTION
**Pull request by stavros-k**
**Description**

⚒️ Fixes  # 

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**

**📃 Notes:**

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning

### CODE REVIEW

File: Plans/blacklist.txt [hunk 1]
```diff
@@ -33,3 +33,10 @@ amd
 ApacheGuacamole
 ApacheGuacamoleNoMariaDB
 collabora
+Caddy
```
Comment (Ornias1993) [inline Plans/blacklist.txt:36]: WHy is caddy here?

File: Plans/blacklist.txt [hunk 2]
```diff
@@ -33,3 +33,10 @@ amd
...
+CalibreWeb
+CloudFlare
+CloudflareDDNS
```
Comment (Ornias1993) [inline Plans/blacklist.txt:39]: Why is cloudflareddns considered permanently blocked?

Comment (stavros-k) [reply Plans/blacklist.txt:39]: It's already couple times in the list

File: Plans/blacklist.txt (hunk 1 above)
Comment (stavros-k) [reply Plans/blacklist.txt:36]: I thought it was a reverse proxy, looking it again looks like it's a web server?

Comment (Ornias1993) [reply Plans/blacklist.txt:36]: yes it is :)

Comment (Ornias1993) [reply Plans/blacklist.txt:36]: But you can move it to req. for other reasons :)

File: Plans/blacklist.txt (hunk 2 above)
Comment (Ornias1993) [reply Plans/blacklist.txt:39]: What do you mean?

Comment (stavros-k) [reply Plans/blacklist.txt:39]: image

It's the same thing with "similar" name

**Pull request merged**